synapse/app/homeserver.py

```diff
@@ -307,6 +307,7 @@ def run_startup_checks(self, db_conn, database_engine):
 # Gauges to expose monthly active user control metrics
 current_mau_gauge = Gauge("synapse_admin_mau:current", "Current MAU")
 max_mau_gauge = Gauge("synapse_admin_mau:max", "MAU Limit")
+reserved_mau_gauge = Gauge("synapse_admin_mau:reserved", "Reserved real MAU users")
```

Comment:
Can we call this something other than `reserved`, since I'd expect it to match the number of threepids in the DB.